molo/surveys/forms.py

```diff
@@ -20,6 +20,17 @@
 CHARACTER_COUNT_CHOICE_LIMIT = 512
 
 
+class SurveyBaseForm(BaseForm):
+
+    def pk_cleaned_data(self, fields):
```

Comment:
Would we be able to put this in the survey model? We could use the `get_fields` method and pass in the `cleaned_data`
That way you wouldn't have to overwrite the form class. Or am I missing something?

Reply:
@KaitCrawford I am not sure how you mean by this, but I have since tried to clean up the code